### DISCUSSION TITLE
Configure multiple triggers

### DISCUSSION THREAD

**Discussion by herrdragon** [edgexfoundry/community#164] [General]
Hi Team,

How to configure to have multiple triggers working on a app-service like 
[Trigger]
Type = "http, edgex-messagebus, external-mqtt"
Is there any examples out there?

Thank you!

**Comment by lenny-goodell** [comment]
This is not supported. Only one Trigger type is allowed.

**Reply by herrdragon** [reply]
Ok, thank you
